php/Terminus/Models/Collections/Backups.php

```diff
@@ -130,6 +166,44 @@ function($backup) {
   }
 
   /**
+   * Sets an environment's regular backup schedule
+   *
+   * @param [integer] $day_number A numerical of a day of the week
+   * @return [boolean] True if operation was successful
+   */
+  public function setBackupSchedule($day_number) {
+    $daily_ttl   = 691200;
+    $weekly_ttl  = 2764800;
```

Comment:
I wonder if we should set these as Constants at the top of the file. Or not. They haven't changed in the 2 years I've been here.... but you never know :-)